examples/s3-backup-stream.js

```diff
@@ -169,8 +169,7 @@ function backupToS3(sourceUrl, s3Client, s3Bucket, s3Key, shallow) {
  */
 function s(originalUrl) {
   var parts = new url.URL(originalUrl);
-  delete parts.auth;
-  return url.format(parts);
```

Comment:
Copyright update for this file

Reply:
Done in e127bfe20c4af9e7ce223a8788ef9a2c6a6422c5